flashinfer/fused_moe/sm90_nvfp4_repack.py

```
@@ -0,0 +1,1665 @@
+    if len({tensor.device for tensor in tensors.values()}) != 1:
+        raise ValueError("v3 tensors must share a device")
+
+    experts, padded_n, padded_k = view.manifest.padded_shape
+    expected_payload = (
+        experts,
+        padded_k // NVFP4_SM90_TILE_K,
+        padded_n // NVFP4_SM90_TILE_N,
+        NVFP4_SM90_TILE_N,
+        NVFP4_SM90_TILE_K // 2,
+    )
+    expected_scales = expected_payload[:-1] + (NVFP4_SM90_TILE_K // 16,)
```

Ruff (0.16.1)

[warning] Consider `(*expected_payload[:-1], NVFP4_SM90_TILE_K // 16)` instead of concatenation

Replace with `(*expected_payload[:-1], NVFP4_SM90_TILE_K // 16)`

(RUF005)